colossalai/booster/plugin/gemini_plugin.py

```diff
@@ -10,6 +10,7 @@
 from torch.optim import Optimizer
 from torch.optim.lr_scheduler import _LRScheduler as LRScheduler
 from torch.utils.data import DataLoader
+from torch.nn.parallel import DistributedDataParallel as DDP
```

Comment:
Don't use DDP. There sychronization logic may have conflicts

Reply:
Updated. DDP is removed. I ran forward within the 'no_sync' context, It seems that DDP is redundant.